src/lib/mdns/Resolver_ImplMinimalMdns.cpp

```diff
@@ -238,11 +223,26 @@ void PacketDataReporter::OnResource(ResourceType type, const ResourceData & data
         }
         else if (mDiscoveryType == DiscoveryType::kOperational)
         {
-            OnOperationalSrvRecord(data.GetName(), srv);
+            // Ensure this is our record.
+            if (HasQNamePart(data.GetName(), kOperationalServiceName))
```

Comment:
previous code was checking that the suffix was fully _chip._tcp.local.

updated code seems to check only that "_chip" occurs somewhere in the qname. Is this ok and sufficient? Maybe a comment of "note: this does not fully validate path" may be useful if we at some point decide we really need to fully validate the path.

Reply:
We should fix this too.